Sources/Function.swift

```diff
@@ -70,14 +83,8 @@ func prettyDictionaryString<K, V>(_ d: [K: V], debug: Bool) -> String {
 func valueString<T>(_ target: T, debug: Bool) -> String {
     let mirror = Mirror(reflecting: target)
 
-    if mirror.children.count == 1, !debug {
-        guard let value = mirror.children.first?.value else { preconditionFailure() }
-        if let string = value as? String {
-            return "\"\(string)\""
-        } else {
-            return String(describing: value)
-        }
-    }
+    // Note: this function support Optional type currently that inclue 1 children.
```

Comment:
```suggestion
    // Note: this function currently supports Optional type that includes a child.
```

Reply:
I wonder what is returned as `displayStyle` if `x` is `Any` type. `Class` or `Struct`?

Reply:
This function `valueString()` is assume limited argument that has no fields (`Mirror#children) such as `String` or `Int` and other Basic type in Swift. This function also supports `Optional` type, but `Optional` type has one field (`Mirror#children.count == 1`). This comment is explains it.

Note:
I'll write test for this function at codes are organized.